linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -885,6 +945,70 @@ impl ClientWrapper {
     }
 }
 
+impl Drop for ClientWrapper {
+    fn drop(&mut self) {
+        use std::process::Command as SyncCommand;
+
+        let binary_path = self
+            .binary_path
+            .lock()
+            .expect("Threads should not panic while holding a lock to `binary_path`");
+
+        let Some(binary_path) = binary_path.as_ref() else {
+            // The command binary was never resolved, so it is assumed to have never been called
+            // and therefore no chains need to be removed.
```

Comment:
Alternatively, this comment could be replaced by a short warning.

Reply:
Done.